soroban-sdk/src/tests/contract_docs_enum.rs

```diff
@@ -0,0 +1,46 @@
+use crate as soroban_sdk;
```

Comment:
I'm not strictly against creating multiple test files, but just wonder if there is something preventing us from consolidating the tests into a single file.

Reply:
Nothing prevents it. Anything we can do in a file we can do inside a mod inside one file. I'll try that before merging to see if it's too overwhelming.

Reply:
Consolidated the tests in one big file: 3d6c7f63